test/widgets/inbox_test.dart

```diff
@@ -64,7 +79,9 @@ void main() {
       users: [eg.selfUser, eg.otherUser, eg.thirdUser],
       unreadMessages: [
         eg.streamMessage(stream: stream1, topic: 'specific topic', flags: []),
+        ...generateStreamMessages(stream1, 10),
         eg.streamMessage(stream: stream2, flags: []),
+        ...generateStreamMessages(stream2, 40),
         eg.dmMessage(from: eg.otherUser, to: [eg.selfUser], flags: []),
         eg.dmMessage(from: eg.otherUser, to: [eg.selfUser, eg.thirdUser], flags: []),
       ]);
```

Comment:
I still used the `setupVarious` method for the new tests as I figured out there was no need for two additional streams. I just added additional messages to the existing streams. If you still believe it has made it complicated, please let me know so that I set them up separately for the new tests!

Reply:
Cool. I think this version of `setupVarious` is fine.